docs/guides/expectations/create_expectations_overview.md

```diff
@@ -3,4 +3,67 @@ title: "Create Expectations: Overview"
 ---
 # [![Create Expectations Icon](../../images/universal_map/Flask-active.png)](./create_expectations_overview.md) Create Expectations: Overview
 
-** This is a placeholder until content is created, so that corresponding links don't cause docusaurus to break **
+import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+
+<!--Use 'inactive' or 'active' to indicate which Universal Map steps this term has a use case within.-->
+
+<UniversalMap setup='inactive' connect='inactive' create='active' validate='inactive'/>
+
+:::note Prerequisites
+- Completing [Step 3: Create Expectations](/docs/tutorials/getting_started/create_your_first_expectations) of the Getting Started tutorial is recommended.
+:::
+
+Creating Expectations is an integral part of Great Expectations.  By the end of this step, you will have created an Expectation Suite containing one or more Expectations which you will use when you Validate data.
+
+## The Create Expectations process
+
+There are a few workflows you can potentially follow when creating Expectations.  These workflows represent various ways of creating Expectations, although they converge in the end when you will save and test those Expectations.  Of the three potential ways to create Expectations, two are recommended in particular.
+
+First is the interactive work flow.  In this workflow, you will be working in a Python interpreter or Jupyter notebook.  You will use a Validator and its `expect()` method to define Expectations in an Expectation Suite, and when you have finished you will save that Expectation Suite into your Expectation Store.  A more thorough overview of this workflow, and a link to an in depth guide on it, can be found in this document's section on [creating Expectations interactively](#creating-expectations-automatically).
```

Comment:
Hmm - calling it the `expect()` method isn't quite accurate. Maybe something like the below? (It's a little clunky, so feel free to adjust).
```suggestion
First is the interactive workflow.  In this workflow, you will be working in a Python interpreter or Jupyter notebook.  You will use a Validator and call expectations as methods on it to define Expectations in an Expectation Suite, and when you have finished you will save that Expectation Suite into your Expectation Store.  A more thorough overview of this workflow, and a link to an in depth guide on it, can be found in this document's section on [creating Expectations interactively](#creating-expectations-automatically).
```